crt_portal/cts_forms/migrations/0054_report_public_id.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.10 on 2020-03-09 22:44
+import random
+
+from django.db import migrations, models
+
+
+def add_salt():
```

Comment:
This looks to be identical to `signals.salt`, let's define in one place and call where needed